scripts/migrations/index.mjs

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env node
+
+import yargs from 'yargs'
+import { hideBin } from 'yargs/helpers'
+
+import { config } from 'dotenv-safe'
+config()
+
+import { spawnSync } from 'child_process'
+
+const {
+  DATO_FULL_ACCESS_API_TOKEN_RWS_VIEWER,
+  DATO_FULL_ACCESS_API_TOKEN_DELTARES_VIEWER,
+} = process.env
+
+const apiTokens = [
+  DATO_FULL_ACCESS_API_TOKEN_DELTARES_VIEWER,
+  DATO_FULL_ACCESS_API_TOKEN_RWS_VIEWER,
+]
+
+yargs(hideBin(process.argv))
+  .command('new', 'Create a new migration', (yargs) => {
+    yargs.option('name', {
+      describe: 'Name of the migration',
+      demandOption: true,
+      requiresArg: true,
+      type: 'string',
+    })
+  }, ({ name }) => {
+    const { stdout, stderr } = spawnSync('npx', [
+      'datocms',
+      'migrations:new',
+      `'${ name }'`,
+      `--api-token=${ apiTokens[0] }`,
+    ])
+
+    console.log(stderr.toString())
+    console.log(stdout.toString())
```

Comment:
This can be removed right?

Reply:
No, that logs the output from DatoCMS CLI back to the user.